Automattic/jetpack
# VideoPress: reorganize and improve URL helpers

This PR improves the regex to pick the `guid` from a VideoPress URL. Also, moves the `url/` into `utils/` folder at `client/` level.

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: reorganize and improve URL helpers

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* All changes happen in the Block Editor context, so
* Edit a post
* Paste the following URL (for instance) to get a core/embed videopress instance: `https://videopress.com/v/SEhvBzm2`
* Convert the instance to a VideoPress video block
* Change some video attributes using the block sidebar
* Confirm it works as expected
